_blog.yml

```
@@ -6039,4 +6039,14 @@
     - guide
     - diffusers
     - quantization
-    - flux
\ No newline at end of file
+    - flux
+
+- local: python-tiny-agents 
+  title: "Tiny Agents in Python: a MCP-powered agent in ~70 lines of code"
+  author: celinah
+  thumbnail: /blog/assets/python-tiny-agents/thumbnail.png
+  date: May 22, 2025
```

```suggestion
  date: May 23, 2025
```

Reminder to update the date